Gems/ROS2/Code/Source/Manipulation/FollowJointTrajectoryActionServer.cpp

```diff
@@ -124,7 +132,7 @@ namespace ROS2
     {
         AZ_TracePrintf("FollowJointTrajectoryActionServer", "Goal accepted\n");
         m_goalHandle = goalHandle;
-        m_goalHandle->execute();
```

Comment:
We trigger here denied transition : 
```
terminate called after throwing an instance of 'rclcpp::exceptions::RCLError'
  what():  goal_handle attempted invalid transition from state EXECUTING with event EXECUTE, at ./src/rcl_action/goal_handle.c:95
```
It is used only when goals are in the queue.